docs/api.rst

```diff
@@ -12122,3 +12119,50 @@ Custom Assets
    :statuscode 200: Custom asset types retrieved successfully.
    :statuscode 409: No user is currently logged in. Check error message.
    :statuscode 500: Internal rotki error.
+
+
+Events Details
+================
+
+.. http:get:: /api/(version)/history/events/details
+
+   Doing a GET on this endpoint will return the details of a history event.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/assets/custom HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {"event_identifier": "0x66e3d2686193e01a4fbf0f598872236402edfe2f4efad84c4f6cdc753b8c78e3", "sequence_index": "222"}
+
+   :reqjson int[optional] event_identifier: The identifier of the event to be queried.
+   :reqjson string[optional] sequence_index: The sequence_index of the event to be queried.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "sub-swaps": [
+                  {"amount_in": "100.0", "amount_out": "0.084", "from_asset": A_SUSHI.identifier, "to_asset": A_ETH.identifier},  # noqa: E501
+                  {"amount_in": "0.084", "amount_out": "98.2", "from_asset": A_ETH.identifier, "to_asset": A_USDT.identifier},  # noqa: E501
+              ],
+          },
+          "message": ""
+      }
+
+   :resjson object result: A dictionary with the details. If there is no details, an empty dictionary is returned.
+   :resjson list sub-swaps: A list with the details of each sub-swap. Each entry contains the following keys: amount_in, amount_out, from_asset, to_asset.
+
+   :statuscode 200: The details were returned successfully.
+   :statuscode 400: Provided JSON is in some way malformed.
+   :statuscode 404: There is no event with the provided event_identifier and sequence_index.
```

Comment:
```suggestion
   :statuscode 404: There is no event with the provided event_identifier or sequence_index.
```

Reply:
Why this change? The events are matched by event_identifier + sequence_index

Reply:
it was due to how the docs were. If it was a mistaken then it is an `and`. If it was optional then is `or`